iOS/Source/InformationController.swift

```diff
@@ -9,6 +9,13 @@ class InformationController: UIViewController {
 
     var messageLabelHeightAnchor: NSLayoutConstraint?
 
+    lazy var webButton: UIButton = {
+        let button = UIButton(withAutoLayout: true)
+        button.addTarget(self, action: #selector(didClickWebButton), for: .touchUpInside)
```

Comment:
Can we have `didSelectLinkButton` instead? Having anything called click in iOS doesn't feel right.